go.mod

```diff
@@ -6,8 +6,6 @@ toolchain go1.22.7
 
 replace (
 	github.com/docker/distribution => github.com/docker/distribution v2.8.2+incompatible // rancher-machine requires a replace is set
-	// Temporary. will be removed shortly
-	github.com/rancher/rancher/pkg/apis => github.com/rohitsakala/rancher/pkg/apis v0.0.0-20241126182546-542381f93109
```

Comment:
The new pkg/apis(L36)  need to be bumped  from `github.com/rancher/rancher/pkg/apis v0.0.0-20241113133627-598640d1556c` to `github.com/rancher/rancher/pkg/apis v0.0.0-20241127174121-c051d99dcded`